test/babel/css-prop.test.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-env jest */
+import * as babel from 'babel-core'
+import plugin from '../../src/babel'
+import * as fs from 'fs'
+
+jest.mock('fs')
+
+fs.existsSync.mockReturnValue(true)
+
+describe('babel css prop', () => {
+  test('basic', () => {
+    const basic = '(<div className="a" css={`color: brown;`}></div>)'
+    const { code } = babel.transform(basic, {
+      plugins: [plugin],
+      filename: __filename,
+      babelrc: false
+    })
```

Comment:
You need to add this to test that the css files were written correctly. 
```js
expect(fs.writeFileSync).toHaveBeenCalledTimes(1)
expect(fs.writeFileSync.mock.calls[0][1]).toMatchSnapshot()
```
You need to increment the `toHaveBeenCalledTimes` by one each time and for this `fs.writeFileSync.mock.calls[0][1]` increment the first number(the 0)